model/src/main/scala/ch.epfl.scala.index.model/release/ScalaTarget.scala

```diff
@@ -80,6 +88,47 @@ object ScalaJvm {
     }
     ScalaJvm(binaryVersion)
   }
+
+  val `2.10`: BinaryVersion = MinorBinary(2, 10)
+  val `2.11`: BinaryVersion = MinorBinary(2, 11)
+  val `2.12`: BinaryVersion = MinorBinary(2, 12)
+  val `2.13`: BinaryVersion = MinorBinary(2, 13)
+
+  private def stableBinaryVersions = Set(`2.10`, `2.11`, `2.12`, `2.13`)
+
+  def isValid(version: BinaryVersion): Boolean =
+    stableBinaryVersions.contains(version)
+}
+
+object ScalaJs {
+  val `0.6`: BinaryVersion = MinorBinary(0, 6)
+  val `1.x`: BinaryVersion = MajorBinary(1)
+
+  private def stableBinaryVersions = Set(`0.6`, `1.x`)
+
+  def isValid(version: BinaryVersion): Boolean =
+    stableBinaryVersions.contains(version)
+}
+
+object ScalaNative {
+  val `0.3`: BinaryVersion = MinorBinary(0, 3)
+  val `0.4.0-M2`: BinaryVersion = PreReleaseBinary(0, 4, Some(0), Milestone(2))
+
+  def isValid(version: BinaryVersion): Boolean = {
+    version == `0.3` || version >= `0.4.0-M2`
+  }
+}
+
+object SbtPlugin {
+  val `0.13`: BinaryVersion = MinorBinary(0, 13)
+  val `1.0`: BinaryVersion = MinorBinary(1, 0)
+  val `1.x`: BinaryVersion = MajorBinary(1)
```

Comment:
I don't think that sbt uses `1.x` or `1`. They (mistakenly) always use the binary version `1.0`, even though it also applies to all 1.1.0 and following.

Reply:
That's right. I will remove this.
 Thank you for reviewing.